src/Utils/file_utils.py

```diff
@@ -58,23 +58,32 @@ def get_human_verified_results_local_excel(filename):
     except Exception as e:
         raise ValueError(f"Failed to read Excel file at {filename}: {e}")
 
-    df.columns = df.columns.str.strip().str.lower()
-    expected_issue_id = "issue id"
+    # Convert column names to strings first to avoid .str accessor errors with NaN/numeric columns
+    df.columns = df.columns.astype(str).str.strip().str.lower()
+
     expected_false_positive = "false positive?"
 
-    if expected_issue_id not in df.columns:
-        raise KeyError(
-            f"Expected column '{expected_issue_id}' not found in the file. \
-                Found columns: {list(df.columns)}"
-        )
     if expected_false_positive not in df.columns:
         raise KeyError(
             f"Expected column '{expected_false_positive}' not found in the file. \
                 Found columns: {list(df.columns)}"
         )
 
-    ground_truth = dict(zip(df[expected_issue_id], df[expected_false_positive]))
-    logger.info(f"Successfully loaded ground truth from {filename}")
+    # Generate issue IDs automatically (def1, def2, ...) to match Google Sheets behavior
+    ground_truth = {}
+    for idx, row in df.iterrows():
+        issue_id = f"def{idx + 1}"  # idx starts at 0, so add 1 to get def1, def2, ...
+        is_false_positive = str(row[expected_false_positive]).strip().lower()
+
+        if is_false_positive not in ALL_VALID_OPTIONS:
+            logger.warning(
+                f"Warning: {issue_id} has invalid value '{is_false_positive}' \
+                    in 'False Positive?' column."
+            )
+
+        ground_truth[issue_id] = is_false_positive
```

Comment:
This section we have duplicated in `get_human_verified_results_google_sheet` as well. Can you extract this part into a private method to avoid code duplication?